pysat/utils/files.py

```diff
@@ -223,7 +232,7 @@ def parse_fixed_width_filenames(files, format_str):
 
 
 def parse_delimited_filenames(files, format_str, delimiter):
-    """Parse list of files, extracting data identified by format_str.
+    """Extract time/version info from a list of files with a varied name width.
```

Comment:
Technically delimited files don't have to have a variable name width.

Reply:
Same note about including custom keywords.

Reply:
```suggestion
    """Extract specific info from a list of files with a fixed name width using delimiters.
```